exs/mundo_1/javascript/005.js

```diff
@@ -0,0 +1,15 @@
+/*
+Desafio 005
+
+Problema: Faça um programa que leia um número inteiro e mostre na tela o seu sucessor e seu antecessor
+
+Resolução do problema:
+*/
+
+const input = require('readline-sync').question
+
+let num = Number(input('Insira um número inteiro: '))
+let intNum = Math.round(num)
+
+console.log(`O numero digitado foi ${intNum}`)
```

Comment:
Corrija o erro ortográfico
```suggestion
console.log(`O número digitado foi ${intNum}`)
```